gpx-docs/docs/commands/Guard.md

```
@@ -0,0 +1,71 @@
+
+### Activate the guard
+
+Navigate to your repository and run the command.
+
+```bash
+cd ~/projects/company-app
+gpx guard
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)